paderbox/visualization/plot.py

```diff
@@ -846,3 +853,197 @@ def seq2seq_alignment(alignment, targets=None, decode=None, ax=None,
     ax.set_xlabel('frame index')
     ax.set_ylabel('decode')
     return ax
+
+
+@create_subplot
+def barh(y, width, height=0.8, left=None, *, ax=None, scale=1, **kwargs):
+    """
+    A wrapper around `plt.barh` that can also handle
+    `pb.array.interval.ArrayInterval`s.
+
+    Args:
+        y: The `y` coordinate(s). Can be a scalar (all bars are drawn at the
+            same y coordinate) or array-like of length `len(width)` (each
+            element of `width` has its own y coordinate)
+        width: Scalar, array-like, `pb.array.interval.ArrayInterval` or list of
+            `pb.array.interval.ArrayInterval`.
+        height:
+        left:
+        ax:
+        scale:
```

Comment:
should we copy the args definition from plt.barh?

Reply:
Yes, we could do that. Or write "see plt.barh".